lib/smart_answer_flows/marriage-abroad/outcomes/countries/philippines/_opposite_sex.erb

```diff
@@ -2,47 +2,15 @@
 Contact the [Civil Registrar](https://psa.gov.ph/lcr-directory) where you want to get married to find out about local marriage laws, including what documents you’ll need.
 
 ## Prove you’re free to get married
-You need to swear an affirmation or affidavit that says you’re legally allowed to get married.
+You need to swear an affidavit that says you’re legally allowed to get married. You must do this through a local notary public in the Philippines. 
 
-This costs <%= format_money calculator.consular_fee(:affidavit_or_affirmation_for_marriage) %> per person. You can pay:
+Contact the [Civil Registrar](https://psa.gov.ph/lcr-directory) to find a local notary public. Check with them what you need to do and what documents you’ll need to bring when you go to swear your affidavit.
+You can download an [affidavit template](/government/publications/marriage-in-the-philippines) to use.
```

Comment:
@CarolineCMAtkinson Can I check if this line was intended to be a separate paragraph, or another sentence at the end?

Reply:
Hi Bruce, do you mean the line about downloading the template? 

I originally had it on a separate line when working from a google doc - but then thought it looked neater just to include at the end of the paragraph when I put it in Github. 

The stakeholder has signed off on both versions - but I think it's better just at the end of the paragraph. Thanks for your help.

PS I'm on leave now so will check a bit later in case you reply...

Reply:
@CarolineCMAtkinson Thanks for the information, it was indeed the line about downloading the template.  I'll edit this to put it all on one line (it's currently spread over two lines), so it's clearer to anyone making edits later on that this isn't intended to be a separate paragraph.

Reply:
Sounds good - thanks v much